### PR TITLE
Fix some warnings

### PR DESCRIPTION
**Pull request by Saibamen**

### CODE REVIEW

**Comment by marticliment** [comment]
This PR had broken code. I would ask you to please use the https://github.com/marticliment/UniGetUI/blob/main/.github/PULL_REQUEST_TEMPLATE.md Pull Request Template, which enforces some steps before publishing a PR

File: src/.editorconfig [hunk 1]
```diff
@@ -37,9 +37,9 @@ dotnet_style_predefined_type_for_locals_parameters_members = true:suggestion
 dotnet_style_predefined_type_for_member_access = true:suggestion
 
 # Require var all the time.
-csharp_style_var_for_built_in_types = true:suggestion
-csharp_style_var_when_type_is_apparent = true:suggestion
-csharp_style_var_elsewhere = true:suggestion
+csharp_style_var_for_built_in_types = false
+csharp_style_var_when_type_is_apparent = false
```
Comment (Saibamen) [inline src/.editorconfig:42]: Maybe leave `when_type_is_apparent` as `true:suggestion`?

Comment (marticliment) [reply src/.editorconfig:42]: I will change it in main

**Comment by Saibamen** [comment]
Sorry, I had build problems on my Visual Studio, ARM64 was not found during compilation.

Probably after your commit with `UniGetUI.PackageEngine.Managers.PowerShell7.csproj`, but I'm not sure.

And after installing this package via Visual Studio Installer, I still had this error:
image

But everything was OK for GitHub Actions in this PR

**Comment by marticliment** [comment]
Oh, don't worry then. I faced this issue also, there was a corruption in the solution configuration. The issue should now be fixed

**Comment by Saibamen** [comment]
Current master branch is building for me without any issues, thanks.

**Pull request merged**